realm/realm-library/src/objectServer/java/io/realm/SyncSession.java

```diff
@@ -50,6 +55,8 @@
 public class SyncSession {
     private final static ScheduledThreadPoolExecutor REFRESH_TOKENS_EXECUTOR = new ScheduledThreadPoolExecutor(1);
     private final static long REFRESH_MARGIN_DELAY = TimeUnit.SECONDS.toMillis(10);
+    private final static int DIRECTION_DOWNLOAD = 1;
+    private final static int DIRECTION_UPLOAD = 2;
```

Comment:
Since that would make them part of the public API, I was kinda hoping not doing that?